autotag

```diff
@@ -48,6 +48,19 @@ git fetch --tags
 echo 'Finding most recent tag...'
 
 LAST_TAG=$(git describe --abbrev=0 --tags 2>/dev/null)
+
+# Ensure that tag is not created on same commit as current branch commit
+if [[ ! -z "$LAST_TAG" ]]; then
```

Comment:
It's more elegant to use the not empty check than to check if filled and negate

https://devhints.io/bash

```suggestion
if [[ -n "$LAST_TAG" ]]; then
```